charts/astronomer/templates/registry/registry-scc.yaml

```diff
@@ -0,0 +1,42 @@
+#################################
+### Astronomer Registry Scc   ###
+#################################
+{{- if and .Values.registry.serviceAccount.create .Values.registry.serviceAccount.sccEnabled }}
+apiVersion: security.openshift.io/v1
```

Comment:
@pgvishnuram Did you get this yaml from some online resource? If yes, can you please add the relevant link as a comment?

Reply:
@pgvishnuram ?